src/main/java/frc/robot/commands/HoldVelocity.java

```diff
@@ -57,12 +58,19 @@ public void initialize() {
     // Called every time the scheduler runs while the command is scheduled.
     @Override
     public void execute() {
+        boolean isAtTargetV = false;
+
         if(m_shooter.getVelocity() < m_targetVelocity - 0.2) {
             speedPercent += 0.1;
         }
         else if(m_shooter.getVelocity() > m_targetVelocity + 0.2) {
             speedPercent -= 0.1;
         }
+        else {
+            isAtTargetV = true;
+        }
+
+        SmartDashboard.putBoolean("Shooter at target velocity:", isAtTargetV);
```

Comment:
I'd say don't use colons in your names here. It looked nice in the old smart dashboard, but the colon is implied. It shouldn't really read like a sentence, it should read like a key/value pair. Also, if you use folders you don't need to include "shooter" in it. I'd say a better key would be "Shooter/atTargetVelocity"